reference/promotions.v3.yml

```
@@ -888,6 +905,62 @@ components:
               description: Link to the next page returned in the response.
           description: Pagination links for the previous and next parts of the whole collection.
       description: 'Data about the response, including pagination and collection totals.'
+    DeprecatedPagination:
+      title: Pagination
+      type: object
+      deprecated: true
+      properties:
+        total:
+          type: integer
+          description: Total number of items in the result set.
+        count:
+          type: integer
+          description: Total number of items in the collection response.
+        per_page:
+          type: integer
+          description: 'The amount of items returned in the collection per page, controlled by the limit of items per page parameter.'
```

probably we can remove the quote '
```suggestion
          description: The amount of items returned in the collection per page, controlled by the limit of items per page parameter.
```